lib/chat-api.js

```diff
@@ -22,30 +22,14 @@ const runApiCommand = (arg: ApiCommandArg): Promise<any> => {
     },
   }
 
-  return new Promise((resolve, reject) => {
-    const inputString = JSON.stringify(input)
-    const size = inputString.length
-    execToJson({
-      command: 'keybase',
-      args: ['chat', 'api'],
-      stdinBuffer: Buffer.alloc(size, inputString, 'utf8'),
-    })
-      .then((o: any) => {
-        let res: any = null
-        if (o && o.result) {
-          res = o.result
-        } else if (o && o.error) {
-          const oError = o.error
-          reject(new Error(oError.message || oError.toString()))
-        } else {
-          reject(new Error(`Unknown error parsing result - no "result" field`))
-        }
-        resolve(res)
-      })
-      .catch((err?: Error) => {
-        reject(err)
-      })
+  const inputString = JSON.stringify(input)
```

Comment:
Not an expert on error handling with promises and async functions, so I'm not sure if we need a try catch for errors here. Since the previous version was just throwing errors up, I thought no but I'm not sure.